File: PHP/Product.php
<?php

class Product{

	
	private $price, $idProduct;


	public function __construct($price = 0, $idProduct = ""){
		$this->price = $price;
		$this->idProduct = $idProduct;
	}

	public function setPrice($price){
		$this->price = $price;
	}

	public function setIdProduct($idProduct){
		$this->idProduct = $idProduct;
	}


	public function getPrice(){
		return $this->price;
	}

	public function getIdProduct(){
		return $this->idProduct;
	}


	public function __destruct(){

	}

}

?>

File: JAVA/Product.java
public class Product {

	private String idProduct;
	private int price;

	public Product(int price, String idProduct){
		this.price = price;
		this.idProduct = idProduct;
	}

	public Product(){

	}

	public void setPrice(int price){
		this.price = price;
	}

	public void setIdProduct(String idProduct){
		this.idProduct = idProduct;
	}

	public int getPrice(){
		return this.price;
	}

	public String getIdProduct(){
		return this.idProduct;
	}

}